docs/reference/transform/overview.asciidoc

```diff
@@ -64,15 +64,13 @@ creates a new index that is dedicated to the transformed data.
 [[transform-performance]]
 ==== Performance considerations
 
-{transforms-cap} perform search aggregations on the source 
-indices then index the results into the destination index. Therefore, a 
-{transform} never takes less time than the cumulated duration of the 
-aggregation that it performs and the indexing process.
+{transforms-cap} perform search aggregations on the source indices then index
+the results into the destination index. Therefore, a {transform} never takes
+less time than the cumulated duration of the aggregation and the indexing
```

Comment:
> never takes less time

This is not just about duration but also about memory and CPU. Perhaps this could say that it never takes less time or uses less resources ..

Reply:
Done, thanks!